crates/ruff_cli/src/args.rs

```diff
@@ -109,15 +109,17 @@ pub struct CheckArgs {
         long,
         value_delimiter = ',',
         value_name = "RULE_CODE",
-        help_heading = "Rule selection"
+        help_heading = "Rule selection",
+        hide_possible_values = true
```

Comment:
Added `hide_possible_values` because we don't want `ruff help check` to list all 4,000 codes.^^